segmatch_ros/package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>cmake_modules</depend>
   <depend>doxygen_catkin</depend>
   <depend>eigen_catkin</depend>
+  <depend>eigen_conversions</depend>
```

Comment:
This dependency is for conversion from eigen transform to geometry_msgs::Transform.
We could do it ourselves if you really don/t want to have that dependency